src/components/MarkdownEditor.js

```diff
@@ -79,8 +79,8 @@ class MarkdownEditor extends React.Component {
 
       const { repo, path, branch } = this.props.match.params;
       const { markdown, sha } = this.state;
-      const contents = btoa(markdown);
-      const response = await putContents(repo, path, contents, sha, branch);
+      const content = markdown;
+      const response = await putContents(repo, path, content, sha, branch);
```

Comment:
Should this be renamed to `putContent`?

Reply:
That's a good question. I renamed it to match GitHub's API – the HTTP method is PUT, the URL is `contents` and the field in the JSON body is called `content`:
<img width="660" alt="Screen Shot 2019-09-28 at 2 23 29 PM" src="https://user-images.githubusercontent.com/361429/65822590-cfbfb880-e1fb-11e9-9950-06ffb94d4b51.png">